sdmetrics/reports/multi_table/diagnostic_report.py

```diff
@@ -73,6 +73,28 @@ def _print_results(self, out=sys.stdout):
         print_results_for_level(out, self._results, 'WARNING')
         print_results_for_level(out, self._results, 'DANGER')
 
+    def _convert_datetimes(self, real_data, synthetic_data, metadata):
```

Comment:
I'd move this one to utils as well, and just call the single table `_convert_datetimes` for each table, so we avoid duplicated code.